app/src/main/java/com/example/triptracker/view/map/RecordScreen.kt

```diff
@@ -408,7 +447,10 @@ fun displayTime(time: Long): String {
   val seconds = TimeUnit.MILLISECONDS.toSeconds(time) % 60
   val minutes = TimeUnit.MILLISECONDS.toMinutes(time) % 60
   val hours = TimeUnit.MILLISECONDS.toHours(time)
-
+  if (seconds < 0 || minutes < 0 || hours < 0) {
```

Comment:
This only checks if the value seen is negative and returns an error but I don't see how it is addressed? If after doing Start -> Pause -> Stop -> Start it displays a negative value then your code just returns an error? Please tell me if I'm missing something. I would just set it back to 0. String.format("%02d:%02d:%02d", 0, 0, 0)